wuzz.go

```diff
@@ -1016,7 +1055,19 @@ func main() {
 
 	initApp(app, g)
 
-	err = app.ParseArgs(g)
+	// load config (must be done *before* app.ParseArgs, as arguments
+	// should be able to override config values). An empty string passed
+	// to LoadConfig results in LoadConfig loading the default config
+	// location. If there is no config, the values in
+	// config.DefaultConfig will be used.
+	app.LoadConfig(configPath)
+
+	err = app.ParseArgs(g, args)
+
+	// Some of the values in the config need to have some startup
+	// behavior associated with them. This is run after ParseArgs so
+	// that command-line arguments can override configuration values.
+	app.InitConfig()
```

Comment:
What is the benefit of calling `InitConfig` here, instead of at the end of `app.LoadConfig()`?